crucible-syntax/src/Lang/Crucible/Syntax/Atoms.hs

```diff
@@ -50,7 +49,7 @@ data Keyword = Defun | DefBlock | DefGlobal
              | Just_ | Nothing_ | FromJust
              | Inj | Proj
              | AnyT | UnitT | BoolT | NatT | IntegerT | RealT | ComplexRealT | CharT | StringT
-             | BitvectorT | VectorT | FPT | FunT | MaybeT | VariantT | RefT
+             | BitvectorT | VectorT | SequenceT | FPT | FunT | MaybeT | VariantT | StructT | RefT
```

Comment:
The `crucible-syntax` README ought to be updated in light of these new types being introduced. (At the very least, the "Types" section.)